### DISCUSSION TITLE
Platform API SDKs

### DISCUSSION THREAD

**Discussion by tbarbugli** [GetStream/protocol#177] [Announcements]
Stream product offering started with Feeds and expanded into Chat. Until now we had one official SDK clients for each popular languages on both products. Starting with video, we decided to change our strategy and create unified Platform API SDKs that expose the API for all products.

If you do not see your language in this list, you can still integrate to our REST API using the [openapi](https://github.com/GetStream/protocol/tree/main/openapi) spec. Before doing that please make sure to reach out to us so we can provide help (and perhaps add your language to this list)

**Comment by tbarbugli** [comment]
Node.js and Python SDK are now available and released (v0.1)

**Comment by steve-sun-me** [comment]
Hey team, thanks for the awesome library. We are trying to integrate with C# which is not available yet. Reaching out due to Before doing that `please make sure to reach out to us so we can provide help (and perhaps add your language to this list)` in the announcements to see if we can get docs for the OpenAPI spec 🙏

**Reply by guerinoni** [reply]
https://github.com/GetStream/protocol/tree/main/openapi

**Comment by TinhHuynh** [comment]
> If you do not see your language in this list, you can still integrate to our REST API using the openapi spec.

@tbarbugli Hi, may I know where the openapi spec is located exactly? 
I can't find it in the Stream repositories or the Internet.

I appreciate any help you can provide.

**Reply by guerinoni** [reply]
https://github.com/GetStream/protocol/tree/main/openapi

**Reply by TinhHuynh** [reply]
Thanks for the response. I will check it out!
